pkgs/applications/terminal-emulators/mlterm/default.nix

```diff
@@ -1,5 +1,6 @@
-{
+args@{
```

Comment:
I don't like this pattern TBH, I prefer using `stdenv'`...

Reply:
I don't like it *much* but this is preferable better to the diff noise caused by replacing all references to `stdenv` with `stdenv'` IMHO.

Reply:
> I don't like it _much_ but this is preferable better to the diff noise caused by replacing all references to `stdenv` with `stdenv'` IMHO.

Isn't it just 1 line? It seems like it should either be this line or the `mkDerivation` line.

Reply:
`stdenv` is usually used in multiple places for `stdenv.is*` guards etc. I didn't check for this particular package; I just apply this pattern because it works everywhere.